docs/fastdds/use_cases/large_data/large_data.rst

```diff
@@ -124,6 +124,27 @@ For receiving sockets, the command is:
 
     $> sudo sysctl -w net.core.rmem_max=12582912
 
+Linux also defines per-socket TCP buffer sizes as triplets:
+
+net.ipv4.tcp_wmem = <min> <default> <max> (TCP send buffer)
+net.ipv4.tcp_rmem = <min> <default> <max> (TCP receive buffer)
+
+The middle value is the default used for most connections. If only the global maxima are raised, sockets may still 
+use a small default and drop data during bursts.
```

Comment:
```suggestion
The middle value is the default used for most connections. If only the global maxima are raised, sockets may still 
use a small default and saturate during bursts.
```

Reply:
If sockets are blocking, as it is our case, the send operation will block instead of drop the data.